examples/multilinetextinput/multilinetextinput/app.py

```diff
@@ -11,6 +12,20 @@ def enable_toggle_pressed(self, widget, **kwargs):
     def readonly_toggle_pressed(self, widget, **kwargs):
         self.multiline_input.readonly = not self.multiline_input.readonly
 
+    def foreground_pressed(self, widget, **kwargs):
+        style = self.multiline_input.style
+        if style.color:
+            del style.color
+        else:
+            style.color = RED
+
+    def background_pressed(self, widget, **kwargs):
+        style = self.multiline_input.style
+        if style.background_color:
+            del style.background_color
+        else:
+            style.background_color = YELLOW
+
```

Comment:
FWIW - I've been using the `colors` app as a testbed for color checks.

Reply:
OK, in that case I'll remove these buttons.